language/vm/src/errors.rs

```diff
@@ -107,7 +108,11 @@ impl VMError {
     pub fn expect_no_verification_errors(self) -> Self {
         match self.status_type() {
             status_type @ StatusType::Deserialization | status_type @ StatusType::Verification => {
-                let message = format!("Unexpected verifier/deserialization error! This likely means there is code stored on chain is unverifiable!\nPrevious error {:?}", &self);
+                let message = format!(
+                    "Unexpected verifier/deserialization error! This likely means there is code \
```

Comment:
Grammar (not new to this change): that last sentence has two verbs. I think it needs a "that": e.g., "...code stored on chain *that* is unverifiable!"